lib/spec.js

```diff
@@ -78,210 +78,338 @@ rules = {
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--name-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-NAME-REQ": {
         "level": "error",
-        "rule": "package.json <code>name</code> is required."
+        "rule": "<code>package.json</code> property <code>\"name\"</code> is required.",
+        "details": "Please add the property <code>\"name\"</code> to your <code>package.json</code>. E.g. <code>{\"name\": \"my-theme\"}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson\" target=_blank><code>package.json</code> documentation</a> to see which properties are required and which are recommended."
     },
     "GS010-PJ-VERSION-SEM": {
         "level": "error",
-        "rule": "package.json <code>version</code> must be semver compliant (e.g. 1.0.0)."
+        "rule": "<code>package.json</code> property <code>\"version\"</code> must be semver compliant.",
+        "details": "The property <code>\"version\"</code> in your <code>package.json</code> file must be semver compliant. E.g. <code>{\"version\": \"1.0.0\"}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--version-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-VERSION-REQ": {
         "level": "error",
-        "rule": "package.json <code>version</code> is required."
+        "rule": "<code>package.json</code> property <code>\"version\"</code> is required.",
+        "details": "Please add the property <code>\"version\"</code> to your <code>package.json</code>. E.g. <code>{\"version\": \"1.0.0\"}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson\" target=_blank><code>package.json</code> documentation</a> to see which properties are required and which are recommended."
     },
     "GS010-PJ-AUT-EM-VAL": {
         "level": "error",
-        "rule": "package.json <code>author.email</code> must be valid."
+        "rule": "<code>package.json</code> property <code>\"author.email\"</code> must be valid.",
+        "details": "The property <code>\"author.email\"</code> in your <code>package.json</code> file must a valid email. E.g. <code>{\"author\": {\"email\": \"hello@example.com\"}}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--author-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-CONF-PPP": {
         "level": "recommendation",
-        "rule": "package.json <code>config.page_per_post</code> is recommended. Otherwise, it falls back to 5."
+        "rule": "<code>package.json</code> property <code>\"config.page_per_post\"</code> is recommended. Otherwise, it falls back to 5.",
```

Comment:
Just found an existing bug - `page_per_post` is the wrong name. Should be `posts_per_page`